src/api/app/controllers/webui/staging/projects_controller.rb

```diff
@@ -10,16 +10,27 @@ class ProjectsController < WebuiController
 
       def create
         authorize @staging_workflow
-        staging_project = @staging_workflow.staging_projects.build(name: params[:staging_project_name])
+
+        staging_project = ::Staging::StagingProject.unscoped
```

Comment:
At this point, the project doesn't have (suposedly) an staging working, so I think we should look in Project instead of doing `Staging::StagingProject.unscoped`

Reply:
If I use `Project` instead of `Staging::StagingProject`, I cannot assign a staging workflow later (on line 26) since the `Project` model doesn't have the `staging_workflow` association.

Reply:
It is possible to assign the `id`. The only difference between a Project and a StagingProject is that the second one has a `staging_workflow`. When you look for it in the first place you don't expect it (necessarily) to have a `staging_workflowp`.

I think those kind of queries will bring problems. We are also missing some validations that may make this query even more complicated.